File: C_Plus_Plus/Easy/9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
      if(x<0){
        return false;
      }
      int n2=x,i;
      long sum=0;
      while(x>0){
        i=x%10;
        sum=(sum*10)+i;
        x=x/10;
      }
      return n2==sum;
    }
};

File: Java/Easy/9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int n=x,digit=0;
        while(n>0){
            digit=digit*10+n%10;
            n/=10;
        }
        if(digit==x)
        return true;
        return false;
    }
}